templates/.rubocop.yml

```diff
@@ -40,6 +40,7 @@ Metrics/BlockLength:
   Exclude:
   - 'spec/**/*.rb'
   - 'config/environments/**/*.rb'
+  - 'config/routes.rb'
```

Comment:
Since routes usually grow anyways, ignoring it from the beginning probably makes sense